src/main/resources/messages.properties

```diff
@@ -18,3 +18,6 @@ match.add.successMessage=Match has added successfully
 match.add.playerPlaceholder=----Select Player----
 match.matches=List of matches
 match.page=Match
+notification.message.declined={0} has declined the request for the match on {1}.
+notification.matchrequest.oneopponent=Please verify that {0} won the match on {1}.
+notification.matchrequest.twoopponents=Please verify that {0} and {1} won the match on {2}.
```

Comment:
CamelCase would make this easier to read so:
`oneOpponent` instead of `oneopponent`.
Same for `twoopponents`.